src/main/java/com/example/bookstore/repository/RoleRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.example.bookstore.repository;
+
+import com.example.bookstore.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+
```

Comment:
Add `@Repository`

Reply:
[This is what I found](https://stackoverflow.com/questions/44069367/repository-not-necessary-when-implementing-jparepository) about this annotation. It is old but `JpaRepository <T, ID>` interface still have @NoRepositoryBean annotation

Reply:
Yeap! That is a great point.